{"name": "Не успешая авторизация", "status": "passed", "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1684321938616, "stop": 1684321938663}, {"name": "Нажать Войти", "status": "passed", "start": 1684321938663, "stop": 1684321938811}, {"name": "Войти как 'qwerty@qwerty.qwerty' 'VTer38CX'", "status": "passed", "start": 1684321941826, "stop": 1684321942028}, {"name": "Проверить что ошибка логина отображается", "status": "passed", "start": 1684321942029, "stop": 1684321942632}], "attachments": [{"name": "log", "source": "a2a41a20-556d-4987-ad27-abf066f1b234-attachment.txt", "type": "text/plain"}], "start": 1684321938616, "stop": 1684321942632, "uuid": "39202fcd-584f-46b8-8c38-a96900cabb3e", "historyId": "8eaf6323d358d700d14848100a31cbc2", "testCaseId": "7b6113008a0e4d8a5bad2235a270375d", "fullName": "tests.test_login.TestLogin#test_login_fail", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "8432-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}